cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinProjectBuilder.java

```diff
@@ -50,10 +53,16 @@ public CristinProject build() {
         cristinProject.setProjectFundingSources(extractFundings(nvaProject.getFunding()));
         cristinProject.setParticipants(extractContributors(nvaProject.getContributors()));
         cristinProject.setCoordinatingInstitution(extractCristinOrganization(nvaProject.getCoordinatingInstitution()));
+        cristinProject.setMethod(extractSummary(nvaProject.getMethod()));
+        cristinProject.setEquipment(extractSummary(nvaProject.getEquipment()));
 
         return cristinProject;
     }
 
+    private String extractMainLanguage(URI language) {
```

Comment:
Do you need to provide the URI argument here?

Reply:
Its what is done in the other extractors, so i did that here as well. I know its not needed since its global, but can change later if you want

Reply:
Wait, i see now im mixing language as parameter and language as nvaProject.getLanguage. Will fix either here or in later PR